### ISSUE TITLE
RL traning curve seems going wroing.

### ISSUE THREAD

**Issue by Serenos** [wadeKeith/DeepThinkVLA#1]
Hi, thank you very much for your great work and for open-sourcing the code!

I’m currently trying to reproduce the results, but I’ve encountered an issue during the RL training phase after SFT. Specifically, both the rollout accuracy and the evaluation accuracy are gradually decreasing, while the entropy is fluctuating upward. I’ve carefully aligned my training configuration with the original one provided in the repository.

Attached below is the training curve for your reference. I’d really appreciate it if you could take a look and offer any insights into what might be going wrong.

<img width="1202" height="464" alt="Image" src="https://github.com/user-attachments/assets/6a2c5673-3789-4e0b-baef-e106a80209d3" />

<img width="1202" height="818" alt="Image" src="https://github.com/user-attachments/assets/9234f7c3-7959-4993-a8e3-8e030318ae6d" />

Thank you in advance!

**Comment by wadeKeith** [comment]
Could you please provide more information? Including some hyperparameters of RL training, now the log seems to indicate the problem might be: 1. Learning rate is too high. 2. Batch size is too small, leading to high variance. 3. Advantage/return scale is too large (not normalized). Additionally, in this situation, you can try using a ref model and add kl_loss to prevent the model from gradually deteriorating.
